### PR TITLE
fix(icons): give collections Icons SUIT classNames

### PR DESCRIPTION
**Pull request by nickgriffin1**
Fixes classname for 3 near brand new icons so that they fit the SUIT naming convention.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @nickgriffin1 has signed the CLA. Thanks for the pull request!

**Comment by alexkrolick** [comment]
I thought we were adding a bdl- or box- prefix to the suit names? @jstoffan 👀

File: src/icons/collections/IconCollections.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const IconCollections = ({
     width = 32,
 }: Props) => (
     <AccessibleSVG
-        className={classNames('icon-collections', className, {
+        className={classNames('IconCollections', className, {
```
Comment (jstoffan) [inline src/icons/collections/IconCollections.js]: Common component class names should be prefixed with `bdl-`, so this would be `bdl-IconCollections`.

Comment (nickgriffin1) [reply src/icons/collections/IconCollections.js]: Ah I was under the impression that only shared variable names and common class names  such as "Button" were prefixed with `bdl`. Will fix.

**Pull request merged**